lib/react_client.dart

```diff
@@ -276,10 +272,8 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
   }
 
   /// Wrapper for [Component.componentWillReceiveProps].
-  void handleComponentWillReceiveProps(
-          Component component,
-          ReactDartComponentInternal nextInternal,
-          InteropContextValue nextContext) =>
+  void handleComponentWillReceiveProps(Component component,
+          ReactDartComponentInternal nextInternal, dynamic nextContext) =>
```

Comment:
I believe all of these changes within `ReactDartInteropStatics` will need to be [moved to `ReactDartInteropStatics2`](https://github.com/cleandart/react-dart/pull/161/files#diff-12ac054bb8b76f35c1035afef1d94396R497) once #161 merges.

The existing API should be left alone so that new context is completely opt-in.

Reply:
Same goes for `handleShouldComponentUpdate`, `handeComponentWillUpdate`, etc.